{
  "common": {
    "state": {
      "off": "Off",
      "on": "On",
      "open": "Open",
      "closed": "Closed",
      "connected": "Connected",
      "disconnected": "Disconnected",
      "locked": "Locked",
      "unlocked": "Unlocked",
      "active": "Active",
      "idle": "Idle",
      "standby": "Standby",
      "paused": "Paused",
      "home": "Home",
      "not_home": "Away"
    },
    "config_flow": {
      "title": {
        "oauth2_pick_implementation": "Pick Authentication Method",
        "via_hassio_addon": "{name} via Home Assistant add-on"
      },
      "description": {
        "confirm_setup": "Do you want to start set up?"
      },
      "data": {
        "email": "Email",
        "username": "Username",
        "password": "Password",
        "host": "Host",
        "ip": "IP address",
        "port": "Port",
        "usb_path": "USB Device Path",
        "access_token": "Access Token",
        "api_key": "API Key"
      },
      "create_entry": {
        "authenticated": "Successfully authenticated"
      },
      "error": {
        "invalid_api_key": "Invalid API key",
        "invalid_access_token": "Invalid access token",
        "cannot_connect": "Failed to connect",
        "invalid_auth": "Invalid authentication",
        "unknown": "Unexpected error"
      },
      "abort": {
        "single_instance_allowed": "Already configured. Only a single configuration possible.",
        "already_configured_account": "Account is already configured",
        "already_configured_service": "Service is already configured",
        "already_configured_device": "Device is already configured",
        "no_devices_found": "No devices found on the network",
        "oauth2_missing_configuration": "The component is not configured. Please follow the documentation.",
        "oauth2_authorize_url_timeout": "Timeout generating authorize url."
      }
    }
  }
}
